www/.htaccess

```diff
@@ -61,14 +63,14 @@ RewriteRule ^.*docs/(\w\w(?:-\w\w)?)/latest$ {{site.baseurl}}/docs/$1/latest/ [R
 #
 #     docs/*/XX/* -> docs/*/YY/*
 #
-{% for redirect in site.data.redirects.version-renames %}RewriteRule ^.*docs/(\w\w(?:-\w\w)?)/{{redirect.old}}/(.*)$ {{site.baseurl}}/docs/$1/{{redirect.new}}/$2 [R=302,L]
+{% for redirect in site.data.redirects.version-renames %}RewriteRule ^.*docs/(\w\w(?:-\w\w)?)/{{redirect[0]}}/(.*)$ {{site.baseurl}}/docs/$1/{{redirect[1]}}/$2 [R=302,L]
```

Comment:
The resulting `.htaccess` file has too many spaces if we add an extra space like this:

    {% for ... %}
    RewriteRule ...
    {% endfor %}

because it will render as:

    RewriteRule ...
    
    RewriteRule ...

We can still refactor it from this:

    {% for ... %}RewriteRule ... 
    {% endfor %}

to this:

    {% for ... %}
    RewriteRule ... {% endfor %}

Which do you think is better?